blog/2025-04-01-network.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Network Team Update
+slug: 2025-04-01-network
+authors: [coot]
+tags: [network]
+hide_table_of_contents: false
+---
+
+## Overview of [sprint 84][sprint-84]
+
+### High-level overview
+
+#### Mithril Development
+
+We continued to cooperate with the Mithril team.  There's a [pull
+request][PR#7] to update the [CIP-0137].
+
+#### Tx-Submission
+
+We continued working on `tx-submission`.  We have an experimental [branch][tx-submission-branch] based
+on the comming `cardano-node-10.3` release which we deployed on mainnet.
+
+#### Peras Workshop
+
+Neil Davis [PNSol] and Marcin Szamotulski participated in the Pares workshop
+organised by Tweag in their Paris office.
+
+#### Performance Improvements
+
+* Karl Kntusson's mux performance [PR][ouroboros-network#5093] was merged. 
+* Marcin Wójtowicz opened a [PR][ouroboros-network#5104] with inbound governor
+  performance improvements.
+
+
+### Pull requests
+#### In review
+
+* [**Inbound governor performance improvement**][ouroboros-network#5104]
+* [**cardano-diffusion sublibrary**][ouroboros-network#5082]
+
+#### Work in Progress
+
+* [**TX Submission Logic**][ouroboros-network#4887]
+* [**Public Network State**][ouroboros-network#5046]
```

Comment:
Also my work on the cardano-sublibrary is still on going. Not sure you want to add that since it's still in review phase

Reply:
ok, I'll move it to the other section.